media-collection/public/migration-strategy.js

```diff
@@ -0,0 +1,142 @@
+const createThirdVideoPlayer = thirdVideoPlayerId => {
```

Comment:
This one is using the legacy media interface to record the events. Maybe a better naming for this file? "legacy-media-interface.js"